clang/lib/Headers/aiev2_srs.h

```diff
@@ -16,199 +16,637 @@ INTRINSIC(v32int8) ssrs(v32acc32 acc, int shft, int sign) {
   return __builtin_aiev2_I256_v32_acc32_srs(acc, shft, sign);
 }
 
+INTRINSIC(v32int8)
+ssrs_conf(v32acc32 acc, int shft, int sign, crsat_t sat, crrnd_t rnd) {
+  crsat_t prev_sat = get_sat();
+  crrnd_t prev_rnd = get_rnd();
+  set_satmode(sat);
+  set_rnd(rnd);
+  v32int8 val = __builtin_aiev2_I256_v32_acc32_srs(acc, shft, sign);
```

Comment:
Super-nit: I'd have a slight preference for calling `ssrs(acc, shft, sign)` instead of our builtin. But at this point, I think it's just personal preference. Ignore me :)

Reply:
> Super-nit: I'd have a slight preference for calling ssrs(acc, shft, sign) instead of our builtin. But at this point, I think it's just personal preference. Ignore me :)

+1